src/components/ToggleSwitch.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import ToggleSwitch from './ToggleSwitch'
+import { render, screen } from 'react-dom'
+
+describe('ToggleSwitch', () => {
```

Comment:
Does this test work? 

It seems like it's missing the "it('does something')" part and an expected result such as ".toBeInTheDocument()"